Uplift/Views/Onboarding/OnboardingView.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  OnboardingView.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 11/18/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class OnboardingView: UIView {
+
+    // MARK: - Private vars
+    private let imageView = UIImageView()
+    private let titleLabel = UILabel()
+    private var tableView: UITableView?
+
+    private var tableData: [String]?
+    private var showingClasses = false
+
+    private let tableViewCellHeight: CGFloat = 60
+    private let tableViewCellSpacing: CGFloat = 14
+
+    // MARK: = Delegation
+    var delegate: (([String]) -> Void)?
+
+    // MARK: - Public vars
+    var favorites: [String] = [] // User's selected favorite gyms/classes
+    var hasTableView = false
+
+    init(image: UIImage, text: String, gymNames: [String]? = nil, classNames: [String]? = nil) {
+        super.init(frame: .zero)
+
+        imageView.image = image
+        imageView.contentMode = .scaleAspectFit
+        addSubview(imageView)
+
+        titleLabel.attributedText = generateAttributedString(for: text)
+        titleLabel.numberOfLines = 3
+        addSubview(titleLabel)
+
+        if let data = gymNames ?? classNames {
+            hasTableView = true
+            tableData = data
+            showingClasses = tableData == classNames
+
+            tableView = UITableView(frame: .zero, style: .plain)
+            tableView?.delegate = self
+            tableView?.dataSource = self
+            tableView?.register(FavoriteCell.self, forCellReuseIdentifier: FavoriteCell.identifier)
+            tableView?.isScrollEnabled = false
+            tableView?.clipsToBounds = false
+            tableView?.separatorStyle = .none
+            self.addSubview(tableView!)
```

Comment:
why dont we first initialize it into a `let` constant and then just set `self.tableView = tableView` at the end:
```
let tableView = UITableView(...)
...
self.tableView = tableView
```